release/dashboard/agent_stress_app_config.yaml

```diff
@@ -25,7 +25,6 @@ post_build_cmds:
   - pip install ale-py==0.7
   - pip uninstall importlib-metadata -y
   - pip install importlib-metadata==4.13.0
-  - pip install gym[atari] autorom[accept-rom-license]
```

Comment:
Only one test uses `agent_stress_app_config.yaml` and it doesn't do anything RL related. https://github.com/ray-project/ray/blob/5c110907827e4d77ba7ff91e7a3c4722bf038290/release/release_tests.yaml#L4815